ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -78,24 +80,27 @@ metadata:
             },
             "resources": {
               "limits": {
-                "cpu": "250m",
-                "memory": "300Mi"
+                "cpu": "350m",
+                "memory": "400Mi"
               },
               "requests": {
-                "cpu": "100m",
-                "memory": "200Mi"
+                "cpu": "200m",
+                "memory": "300Mi"
               }
+            },
+            "rbac": {
+              "pspEnable": false
             }
           },
           "status": {}
         }
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.11.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     operatorframework.io/suggested-namespace: avi-system
-    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:378c4bb64cfff509193930b7bdf981c299e306c4bebf6f5f3197255c5c0e38dd
-  name: ako-operator.v1.6.3-stable
+    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:c556d055a9d6773c9a41655051f831dc83e548ee8d505a3794f1590fa6b8ce32
+  name: ako-operator.v1.8.1-stable
```

Comment:
@DixitAakash: name can be just `ako-operator.v1.8.1`

Reply:
This is the oldest version of Openshift, staring which AKO Operator will be supported. So, this annotation **com.redhat.openshift.versions: v4.6** means that AKO Operator will be uploaded to, and be available on Operator Hub of Openshift version v4.6 to v4.11 (latest version).

Reply:
name changed to **ako-operator.v1.8.1**